web/src/views/BuiltinComponents/RightClick/index.vue

```diff
@@ -0,0 +1,38 @@
+<script setup lang="ts">
+import type { Menu } from '@/components/DRightClick/private'
+
+const menu: Menu[] = reactive([
+  { label: '菜单1', key: 'handle' },
+  { label: '菜单2', key: 'handle' },
+  { label: '菜单3', key: 'handle' },
+  { label: '菜单4', key: 'handle' }
+])
+const handleClick = (e: Menu) => {}
```

Comment:
```suggestion

const handleClick = (e: Menu) => {}
```
空格，handleClick 中可以考虑用 useMessage 弹出点提示，给用户点反馈